src/index.html

```diff
@@ -11,12 +11,95 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo_img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="moyo_menu">
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              data-qa="hover"
+              href="#"
+              class="menu_link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="menu-item">
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading.